project_double_alias/models/project_project.py

```diff
@@ -23,11 +24,14 @@ def _create_alias_issue(self):
         alias_obj = self.env['mail.alias'].with_context(
             alias_model_name='project.issue',
             alias_parent_model_name=self._name)
+        alias_defaults = {'project_id': self.id}
+        if self.env['project.issue']._fields.get('analytic_account_id'):
```

Comment:
Put a comment saying that this is if you have installed `project_issue_sheet` module.